src/Orleans.Core/SystemTargetInterfaces/ISiloControl.cs

```diff
@@ -21,9 +20,6 @@ internal interface ISiloControl : ISystemTarget, IVersionManager
         Task<DetailedGrainReport> GetDetailedGrainReport(GrainId grainId);
 
         Task UpdateConfiguration(string configuration);
-
-        /// <summary>Load and initialize newly added stream providers. Remove providers that are not on the list that's being passed in.</summary>
-        Task UpdateStreamProviders(IDictionary<string, ProviderCategoryConfiguration> streamProviderConfigurations);
```

Comment:
I think dynamic adding stream provider feature depends on this method, are we removing that feature ? 
No objection on removing it, just want to confirm that's the intension

Reply:
Yup.  Runtime addition/removal of providers is being dropped.